src/pg_to_evalscript/javascript_processes/between.js

```diff
@@ -0,0 +1,49 @@
+function between(arguments) {
+  const isBetween = (x, min, max, exclude_max) => {
+    let result = x >= min && x <= max;
+    if (exclude_max) {
+      result &= x < max;
+    }
+    return result;
+  };
+
+  const { x, min, max, exclude_max = false } = arguments;
+
+  if (x === null) {
+    return null;
+  }
+
+  if (x === undefined) {
```

Comment:
These checks if a required argument is defined are repeated very often, so it should perhaps be considered to move it to a function in `common`. 

(we did actually have something similar in the old python definitions of processes https://git.sinergise.com/team-6/openeo-sentinelhub-python-driver/-/blob/d6e89c3bc5a6a076406f88622533407e5bf4fea1/workers/process/eq.py#L15)

FFTR

Reply:
I intend to do this in separate PR